### PR TITLE
Feature/upgrade libraries

### PR DESCRIPTION
**Pull request by bigsergey**

### CODE REVIEW

**Comment by mikeqcp** [comment]
@bigsergey It fails on initial build now. You removed `SpritesmithPlugin` from `webpack.base.babel.js`, but you still import generated `*.scss` files in `main.scss` which results in trying to require unexisting files.

**Comment by bigsergey** [comment]
@mikeqcp I removed SpritesmithPlugin by mistake. I have fixed it, please check if it works now.

File: app/modules/maintainers/__tests__/maintainers.sagas.spec.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { expect } from 'chai';
 import { fromJS } from 'immutable';
 import { sandbox } from 'sinon';
 
-import maintainersSaga, {} from '../maintainers.sagas';
+import maintainersSaga, { } from '../maintainers.sagas';
```
Comment (pziemkowski) [inline app/modules/maintainers/__tests__/maintainers.sagas.spec.js]: Can you remove empty brackets?

**Review by pziemkowski — CHANGES_REQUESTED** [review]
Are you sure everything works in production version? IntlProvider is most likely blocking rendering of its children when URL changes, because no props have changed

**Review by pziemkowski — APPROVED** [review]

**Pull request merged**